keras/backend/tensorflow_backend.py

```diff
@@ -190,6 +210,15 @@ def _convert_string_dtype(dtype):
 
 
 def _to_tensor(x, dtype):
+    """Convert the input 'x' to a tensor of type 'dtype'.
```

Comment:
Code delimiters \` would be more appropriate than string quotes here, for `x` and `dtype`.